lib/yt/video_audit/youtube_association.rb

```diff
@@ -0,0 +1,25 @@
+require 'yt/video_audit/base'
+
+module Yt
+  module VideoAudit
+    # Count how many videos (out of base number of recent videos) have
```

Comment:
I see what you mean. Maybe we can just say "Count how many recent videos have..." in all these descriptions.